src/main/java/com/hongliangjie/fugue/MainEntrance.java

```diff
@@ -40,11 +41,12 @@ private static Options createOptions(){
         options.addOption(randomOption);
         options.addOption(expOption);
         options.addOption(logOption);
+        options.addOption(saveModelOption);
 
         return options;
     }
 
-    private static Message parseOptions(Options options, String[] args){
+    protected static Message parseOptions(Options options, String[] args){
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The method parseOptions() has an NPath complexity of 2050](https://www.codacy.com/app/hongliangjie/Fugue/file/2224824709/issues/source?bid=3162994&fileBranchId=3172800#l49)